apps/web/src/pages/integrations/components/NovuProviderBase.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Stack, Text, useMantineColorScheme } from '@mantine/core';
+import { ChannelTypeEnum } from '@novu/shared';
+import { When } from '../../../components/utils/When';
+import { CONTEXT_PATH } from '../../../config';
+import { colors } from '../../../design-system';
+import { Close } from '../../../design-system/icons/actions/Close';
+import { useIntegrationLimit } from '../../../hooks';
+import { LimitBar } from './LimitBar';
+
+interface NovuProviderBaseProps {
+  onClose: () => void;
+  senderInformation?: React.ReactNode;
+  channel: ChannelTypeEnum;
+}
+
+export function NovuProviderBase({ onClose, senderInformation, channel }: NovuProviderBaseProps) {
+  const {
+    data: { limit, count },
+    loading,
+  } = useIntegrationLimit(channel);
+
+  const { colorScheme } = useMantineColorScheme();
+  const isDark = colorScheme === 'dark';
+
+  const logoSrc = isDark ? CONTEXT_PATH + '/static/images/logo-light.png' : CONTEXT_PATH + '/static/images/logo.png';
+
+  const unit = channel === ChannelTypeEnum.EMAIL ? 'emails' : 'messages';
+
+  return (
+    <div
+      style={{
+        position: 'relative',
+        padding: '18px',
+      }}
+    >
+      <CloseButton data-test-id="connection-integration-form-close" type="button" onClick={onClose}>
+        <Close />
+      </CloseButton>
+
+      <Stack spacing={24}>
+        <div>
+          <img src={logoSrc} alt="logo" style={{ maxWidth: 130, maxHeight: 32 }} />
+        </div>
```

Comment:
I think we should make the logo an own component if it doesn't exist yet, with the `maxWidth` and `maxHeight` properties passed and maybe automatically calculating the path and file name of the logo for dark and light versions, so we can standardise the logo naming.